sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -273,4 +274,13 @@ internal object RadarSettings {
         return getSharedPreferences(context).getBoolean(KEY_PERMISSIONS_DENIED, false)
     }
 
+    internal fun updateLastTrackedTime(context: Context) {
+        val timestampSeconds = System.currentTimeMillis() / 1000
```

Comment:
Do you think we should be taking the timestamp of when the track request went through? Currently it is wired to take the time once the track call has completed